ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -665,6 +668,9 @@ open class BottomCommandingController: UIViewController {
         cell.isEnabled = item.isEnabled
         cell.backgroundStyleType = .clear
         cell.backgroundColor = tableViewBackgroundColor
+        if let accessibilityIdentifier = item.accessibilityIdentifier {
```

Comment:
for my own understanding, why do we need to check if identifer is non null before setting it on cell?

Reply:
Isn't it necessary since accessibilityIdentifier is optional?